torch/csrc/jit/frontend/ir_emitter.cpp

```diff
@@ -473,6 +488,9 @@ struct Environment {
           {"ord", std::make_shared<BuiltinFunction>(aten::ord, at::nullopt)},
           {"chr", std::make_shared<BuiltinFunction>(aten::chr, at::nullopt)},
           {"bin", std::make_shared<BuiltinFunction>(aten::bin, at::nullopt)},
+          {"AssertionError",
+           std::make_shared<ExceptionValue>("AssertionError")},
+          {"RuntimeError", std::make_shared<ExceptionValue>("RuntimeError")},
```

Comment:
Hi-level feedback: I think we want to move this stuff into python resolution, instead of special casing these two specific exceptions.